docs/scripts/extensions.py

```diff
@@ -2,9 +2,9 @@
 from typing import List, Optional, Type
 
 import griffe
+from mkdocs_util import get_mkdocstrings_plugin_handler_options, import_object, load_config
```

Comment:
Minor thing, but I'd move this import two lines down back under the `django_components.util.misc`. Because right now it looks as if `mkdocs_util` was an external package.

A convention I follow is:
```py
# Python's standard standard library packages like `os`, `typing`, etc
import os
from typing import Any

# Third-party packages like `django`, or `griffe`
import griffe

# Our own module, like `django_components` or in this case `mkdocs_util`
from django_components.util.misc import get_import_path
```

Reply:
I initially forgot to install pre-commit to sort imports before committing. This bad sorting then triggered pre-commit auto fix here causing probably the hanging merge.